### PR TITLE
CORE-9002: icon props rename & CORE-8802: change chevron rotation direction

### PR DESCRIPTION
**Pull request by dletran**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@geoblink/design-system",
-  "version": "29.11.0",
+  "version": "30.0.0-beta.2",
```
Comment (Navas23) [inline package.json:3]: Don't forget to change this after deploying

**Review by Navas23 — APPROVED** [review]
Nice :ok_hand:

File: src/elements/GeoBorderedBox/GeoBorderedBoxHeader.vue [hunk 2]
```diff
@@ -22,13 +22,13 @@
...
-      @click="emitClose($event)"
+      v-if="isTrailingIconVisible"
+      class="geo-bordered-box-header__trailing-button"
+      @click="emitTrailingIconClick($event)"
     >
       <font-awesome-icon
-        :icon="closeIcon"
-        class="geo-bordered-box-header__close-button-icon"
+        :icon="trailingIcon"
+        class="geo-bordered-box-header__trailing-button-icon"
```
Comment (fofi) [inline src/elements/GeoBorderedBox/GeoBorderedBoxHeader.vue:31]: We are not using this class in the DS?

**Review by fofi — APPROVED** [review]

File: src/elements/GeoBorderedBox/GeoBorderedBoxHeader.vue (hunk 2 above)
Comment (dletran) [reply src/elements/GeoBorderedBox/GeoBorderedBoxHeader.vue:31]: Nope, but we use it in the app in some cases, like ignoring click outside if clicking on the icon

**Pull request merged**